scripts/pi-hole/php/get.php

```diff
@@ -20,14 +20,19 @@
 function getTableContent($listname) {
 	global $db;
 	$entries = array();
-	$results = $db->query("SELECT * FROM $listname");
+	$querystr = implode(" ",array("SELECT ${listname}.*,\"group\".enabled as group_enabled",
+	                              "FROM $listname",
+	                              "LEFT JOIN ${listname}_by_group ON ${listname}_by_group.whitelist_id = ${listname}.id",
+	                              "LEFT JOIN \"group\" ON \"group\".id = ${listname}_by_group.group_id",
+	                              "GROUP BY domain;"));
+	$results = $db->query($querystr);
 
 	while($results !== false && $res = $results->fetchArray(SQLITE3_ASSOC))
 	{
 		array_push($entries, $res);
 	}
 
-	return array($listname => $entries);
+	return array($listname => $entries, "querystring" => $querystr);
```

Comment:
Why is the query string getting returned?

Reply:
This is mainly for debugging purposes. Looking at it again, it contains less information that I initially thought. I will remove this.